app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -1,6 +1,21 @@
 class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
   ManageIQ::Providers::BaseManager::EventCatcher::Runner
-  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze
+
+  def initialize(cfg = {})
+    super(cfg)
+
+    # Accept events with an event category in the keyset and whose event message matches the (optional) pattern value
+    # - key = event category
+    # - value (optional) event message regex
+    @whitelist = {
+      'Hawkular Deployment' => nil
+    }.freeze
+
+    # Some ems events will link to objects in miq inventory. This maps the event category to the object type
+    @types     = {
+      'Hawkular Deployment' => 'MiddlewareServer'
```

Comment:
I think, as this will be used to constantize in `EmsEvent`, you should use a Constant here and stringify it.
`'Hawkular Deployment' => MiddlewareServer.name`
this pattern makes it less error prone for future additions